packages/react-vapor/src/components/infoBox/InfoBox.tsx

```diff
@@ -7,7 +7,8 @@ export class InfoBox extends React.PureComponent<React.HTMLProps<HTMLDivElement>
         return (
             <div
                 {...this.props}
-                className={classNames('text-oxford-blue bg-onahau', this.props.className, styles.infoBox)}
+                className={classNames('text-oxford-blue', this.props.className, styles.infoBox)}
+                style={{backgroundColor: '#ccefff'}}

```

Comment:
I think we should modify InfoBox.scss instead because otherwise we won't be able to override this backgroundColor